packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -122,7 +123,14 @@ const AccountSwitcher = observer(({ history, is_mobile, is_visible }) => {
         closeAccountsDialog();
         if (account_loginid === loginid) return;
         await switchAccount(loginid);
-        Analytics.setAttributes({ account_type: loginid.substring(0, 2) });
+        Analytics.setAttributes({
+            account_type: LocalStore?.get('active_loginid')?.substring(0, 2) ?? 'unlogged',
+            app_id: getAppId(),
+            device_type: isDesktopOs() ? 'desktop' : 'mobile',
+            device_language: navigator?.language || 'en-EN',
+            user_language: getLanguage().toLowerCase(),
+            country: Cookies.get('clients_country') || Cookies.getJSON('website_status'),
+        });
```

Comment:
why are we updating this attrib ute here ? it's already set on app initialisation, same with device_type language and country

Reply:
we have to call `setAttributes` once user change `account_type ` and only one way to that in current analytics implementation - call again with each field
Or, they will be undefined